src/services/ReportService.ts

```diff
@@ -0,0 +1,361 @@
+  const response: Response = await httpLockReport.post({
+    urlReplacements: {
+      '{id}': id.toString(),
+    },
+    entity: {},
+  });
+
+  return response;
+};
+
+/**
+ * Force lock report
+ */
+const forceLockReport = async (id: number): Promise<Response> => {
+  const response: Response = await httpForceLockReport.post({
+    urlReplacements: {
+      '{id}': id.toString(),
+    },
+    entity: {},
+  });
+
+  return response;
+};
+
+/**
+ * Unlock report
+ */
+const unlockReport = async (id: number): Promise<Response> => {
+  const response: Response = await httpUnlockReport.post({
+    urlReplacements: {
+      '{id}': id.toString(),
+    },
+    entity: {},
+  });
+
+  return response;
+};
+
+/**
+ * Get report photos
+ */
+const getReportPhotos = async (reportId: number): Promise<ReportPhotosResponse> => {
+  const response: ReportPhotosResponse = await httpReportPhotos.get<ReportPhotosResponsePayload, ReportPhotosData>(
+    {
+      urlReplacements: {
+        '{id}': reportId.toString(),
+      },
+    },
+    (data) => ({ photos: data?.photos })
+  );
+
+  return response;
+};
+
+/**
+ * Submit report
+ */
+const submitReport = async (id: number): Promise<Response> => {
+  const response: Response = await httpSubmitReport.post({
+    urlReplacements: {
+      '{id}': id.toString(),
+    },
+    entity: {},
+  });
+
+  return response;
+};
+
+/**
+ * upload report photo
+ */
+const uploadReportPhoto = async (file: string): Promise<UploadReportPhotoResponse> => {
+  const response: UploadReportPhotoResponse = await httpUploadReportPhoto.post({ entity: { file: file } });
+
+  if (response.requestSucceeded) {
+    const data: UploadReportPhotoResponsePayload = response.data;
+    response.fileId = data?.fileId;
+  }
+
+  return response;
+};
+
+/**
+ * Get a report photo
+ */
+const getReportPhoto = async (reportId: number, photoId: number): Promise<ReportPhotoResponse> => {
+  const response: ReportPhotoResponse = await httpReportPhoto.get<DownloadReportPhotoResponsePayload, ReportPhotoData>(
+    {
+      urlReplacements: {
+        '{reportId}': reportId.toString(),
+        '{photoId}': photoId.toString(),
+      },
+    },
+    (data) => ({ photo: data?.photo })
+  );
+
+  return response;
+};
+
+/**
+ * Updatea report photo
+ */
+const updateReportPhoto = async (reportId: number, photoId: number, caption: string): Promise<Response> => {
+  return await httpReportPhoto.put({
+    urlReplacements: {
+      '{reportId}': reportId.toString(),
+      '{photoId}': photoId.toString(),
+    },
+    entity: { caption },
+  });
+};
+
+/**
+ * delete report file
+ */
+const deleteReportPhoto = async (reportId: number, fileId: number): Promise<Response> => {
+  return await httpReportPhoto.delete({
+    urlReplacements: {
+      '{reportId}': reportId.toString(),
+      '{fileId}': fileId.toString(),
+    },
+  });
+};
+
+/**
+ * Exported functions
+ */
+const ReportService = {
```

Comment:
lgtm.. would it be worth splitting this into ReportService, ReportLockingService and ReportPhotosService? or is that overkill.. i guess my thoughts are mostly coming from size of service (maybe that's not a valid issue).. cc @charlie-tf @constanzauanini

Reply:
not requesting any changes.. was mostly a question to hear y'alls thoughts

Reply:
Yes, I don't think it's completely necessary, but if we prefer small files I can split it

Reply:
I'm kinda on the side of keeping it all in one. Then there's just one service to interact with. But either way would completely work 🤷

Reply:
Thanks Constanza/Charlie.. one files sounds good 👍